apps/dashboard/src/app/api/engine-status/error-rate/route.ts

```diff
@@ -0,0 +1,73 @@
+import { client, v2 } from "@datadog/datadog-api-client";
+import { NextResponse } from "next/server";
+
+export const POST = async () => {
+  const configurationOpts = {
+    authMethods: {
+      apiKeyAuth: process.env.DD_API_KEY,
+      appKeyAuth: process.env.DD_APP_KEY,
+    },
+  };
+  const configuration = client.createConfiguration(configurationOpts);
+
+  configuration.unstableOperations["v2.queryTimeseriesData"] = true;
+  const apiInstance = new v2.MetricsApi(configuration);
+
+  const params: v2.MetricsApiQueryTimeseriesDataRequest = {
+    body: {
+      data: {
+        attributes: {
+          from: new Date().getTime() - 1000 * 60 * 60 * 24,
```

Comment:
probably would make sense to make this configurable via POST body / query params?

Reply:
We can't allow more than 24 hours right now until the server behind it (prometheus is stable)